spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1194,6 +1194,7 @@ CDI 2.0 introduced the `javax.enterprise.inject.spi.configurator.BeanConfigurato
 
 With `BeanConfigurator` you can perform all the operations defined in <<bean_attributes_configurator>> plus the following:
 
+* Read the bean metadata from a `BeanAttributes` or an `AnnotatedType` with one of its `read()` methods.
```

Comment:
I would opt to incorporate something indicating that the configurator will be initialized with the metadata you read:

>  `read()` methods initiate the configurator with existing `BeanAttributes` or `AnnotatedType` metadata.